src/Numerics/DGMethods/DGFVModel_kernels.jl

```diff
@@ -590,7 +590,7 @@ A finite volume reconstruction is used to construction `Fⁱⁿᵛ⋆`
                             else
                                 T = local_tendency[s]
                             end
-                            tendency[n, s, eH + eV_up] = T
+                            tendency[n, s, eH + eV_up] += T
```

Comment:
Good catch. Does this mean that we also shouldn't scale by beta again here ? So the whole thing can be simplified to just `tendency += local_tendency` regardless of `increment` ?

Reply:
What is beta? modify to tendency += local_tendency

Reply:
`beta` is a scaling factor for the previous tendency. See the description here:
https://github.com/CliMA/ClimateMachine.jl/blob/c693e9e7d85ceff7ba55f71ebdf3ee577d36dc71/src/Numerics/DGMethods/DGModel.jl#L189-L202